### PR TITLE
Remove "Go" reference in welcome

### PR DESCRIPTION
**Pull request by simonwheatley**
We shouldn't refer to `VIP Go`, there is only VIP.

Either:

```
 _    __ ________         ________    ____
| |  / //  _/ __ \       / ____/ /   /  _/
| | / / / // /_/ /______/ /   / /    / /  
| |/ /_/ // ____//_____/ /___/ /____/ /   
|___//___/_/           \____/_____/___/   
                                          
```

…or…

```
_    __ ________  
| |  / //  _/ __ \ 
| | / / / // /_/ /
| |/ /_/ // ____/
|___//___/_/       
```

### CODE REVIEW

File: src/bin/vip.js [hunk 1]
```diff
@@ -41,11 +41,11 @@ const rootCmd = async function() {
...
-		console.log( '  | | / // // /_/ /  / / __/ __ \\' );
-		console.log( '  | |/ // // ____/  / /_/ / /_/ /' );
-		console.log( '  |___/___/_/       \\____/\\____/' );
+		console.log( '   _    __ ________         ________    ____' );
+		console.log( '  | |  / //  _/ __ \       / ____/ /   /  _/' );
+		console.log( '  | | / / / // /_/ /______/ /   / /    / /  ' );
+		console.log( '  | |/ /_/ // ____//_____/ /___/ /____/ /   ' );
+		console.log( '  |___//___/_/           \____/_____/___/   ' );
 		console.log();
 		console.log( '  VIP CLI is your tool for interacting with and managing your VIP Go applications.' );
```
Comment (elazzabi) [inline src/bin/vip.js]: should we do the same here? `VIP Go applications` => `VIP applications`
```suggestion
		console.log( '  VIP CLI is your tool for interacting with and managing your VIP applications.' );
```

Comment (BrookeDot) [reply src/bin/vip.js]: +1

Comment (simonwheatley) [reply src/bin/vip.js]: Thanks, Ahmed!

**Review by elazzabi — APPROVED** [review]
looks good, thank you 🙏

**Pull request merged**